src/arrayMethodIndexOf.test.js

```diff
@@ -9,6 +9,11 @@ test('indexOf2 is added to [].__proto__', () => {
     .toBeInstanceOf(Function);
 });
 
+test(`indexOf2 doesn't call default indexOf`, () => {
+  expect([].indexOf2.toString().indexOf('.indexOf'))
```

Comment:
.indexOf(

Reply:
And what will you do with `.indexOf` in `.includes2` implementation?

Reply:
@mgrinko 
![image](https://user-images.githubusercontent.com/29413892/56043063-2bc57c00-5d45-11e9-823b-849d7988bd28.png)